### PR TITLE
Allow setting priorityClassName

### PR DESCRIPTION
**Pull request by kolorful**
Updated helm chart so that it allows setting priorityClassName

### CODE REVIEW

File: deploy/charts/external-secrets/README.md [hunk 1]
```diff
@@ -59,3 +59,4 @@ The command removes all the Kubernetes components associated with the chart and
 | serviceAccount.create | bool | `true` | Specifies whether a service account should be created. |
 | serviceAccount.name | string | `""` | The name of the service account to use. If not set and create is true, a name is generated using the fullname template. |
 | tolerations | list | `[]` |  |
+| priorityClassName | string | `""` | Pod priority class name. |
```
Comment (ericblackburn) [inline deploy/charts/external-secrets/README.md:62]: This table is in alphabetical order, let's move this to be right after podSecurityContext.

**Comment by knelasevero** [comment]
Hey @kolorful please run `make check-diff` locally and see what changes it makes and push those, please :)

**Comment by kolorful** [comment]
@knelasevero thanks for reviewing. I've fixed the comment in README and ran `make check-diff` locally

```
15:40:24 [ OK ] Finished generating deepcopy and crds
time="2021-08-23T20:40:26Z" level=info msg="Found Chart directories [.]"
time="2021-08-23T20:40:26Z" level=info msg="Generating README Documentation for chart /helm-docs"
15:40:26 [ OK ] Finished generating helm chart files
github.com/external-secrets/external-secrets/pkg/controllers/externalsecret tested by
	github.com/external-secrets/external-secrets/pkg/controllers/externalsecret.test imports
	sigs.k8s.io/controller-runtime/pkg/envtest imports
	sigs.k8s.io/controller-runtime/pkg/internal/testing/addr imports
	io/fs: package io/fs is not in GOROOT (/usr/local/go/src/io/fs)
make: *** [reviewable] Error 1
```

**Pull request merged**